src/modules/flow/netctl/netctl.json

```diff
@@ -0,0 +1,139 @@
+      "in_ports": [
+        {
+          "data_type": "any",
+          "description": "Trigger to connect to network service.",
+          "methods": {
+            "process": "connect_service_name"
+          },
+          "name": "CONNECT"
+        },
+        {
+          "data_type": "any",
+          "description": "Trigger to disconnect to network service.",
+          "methods": {
+            "process": "disconnect_service_name"
+          },
+          "name": "DISCONNECT"
+        }
+      ],
+      "methods": {
+        "open": "open_service_connection",
+        "close": "close_service_connection"
+      },
+      "name": "netctl/service",
+      "options": {
+        "members": [
+          {
+            "data_type": "string",
+            "default": null,
+            "description": "Refer to netctl/list-services on how to get the service name",
+            "name": "name"
+          }
+        ],
+        "version": 1
+      },
+      "out_ports": [
+        {
+          "data_type": "string",
+          "description": "Output network services connection status.",
+          "name": "OUT"
+        }
+      ],
+      "private_data_type": "connection_service_data"
+    },
+    {
+      "category": "netctl",
+      "description": "Get state of a network service. Usually provided by netctl/list-services.",
+      "in_ports": [
+        {
+          "data_type": "any",
+          "description": "Trigger to emit an output packet with current state",
+          "methods": {
+            "process": "get_state"
+          },
+          "name": "TRIGGER"
+        }
+      ],
+      "methods": {
+        "open": "get_state_open",
+        "close": "get_state_close"
+      },
+      "name": "netctl/state",
+      "options": {
+        "members": [
+          {
+            "data_type": "string",
+            "default": null,
+            "description": "Name of network service from netctl/list-services will be fetched.",
+            "name": "name"
+          }
+        ],
+        "version": 1
+      },
+      "out_ports": [
+        {
+          "data_type": "string",
+          "description": "The string with the current state, such as unknown, idle, association, configuration, ready, online, disconnect, remove",
+          "name": "OUT"
+        },
+        {
+          "data_type": "boolean",
+          "description": "Emits true if the device has a verified connection to the internet, false otherwise.",
+          "name": "ONLINE"
+        }
+      ],
+      "private_data_type": "get_state_data"
+    },
+    {
+      "category": "netctl",
+      "description": "Sets the global connection state to offline",
+      "in_ports": [
+        {
+          "data_type": "boolean",
+          "description": "turn on or off the radios",
+          "methods": {
+            "process": "enable_radios"
+          },
+          "name": "ENABLED"
+        }
+      ],
+      "name": "netctl/radios",
```

Comment:
maybe `netctl/radios-offline`.

I'm not sure about port name, maybe input port called `IN` to match `OUT`? Or rename port `OUT` to `ENABLED`?

@bdilly @ceolin @cabelitos ?